repos.bzl

```diff
@@ -18,9 +18,18 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
 load("@bazel_skylib//lib:versions.bzl", "versions")
 load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
 load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
-load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+load(
+    "@io_bazel_rules_go//go:deps.bzl",
+    "go_download_sdk",
+    "go_register_toolchains",
+    "go_rules_dependencies",
+)
+load(
+    "@io_k8s_repo_infra//go:sdk_versions.bzl",
+    "OVERRIDE_GO_VERSIONS",
```

Comment:
Any reason to put this in a different file rather than in here?

Reply:
@fejta -- Mostly long-term maintenance. This puts us on a path to support Go pre-releases and that list is going to get pretty long. I didn't want to clutter up our primary logic with all of the pre-release tuples.